pages/common/linkchecker.md

```diff
@@ -0,0 +1,20 @@
+# linkchecker
+
+> Command line client to check HTML documents and websites for broken links.
+> More information: <https://linkchecker.github.io/linkchecker/>.
+
+- Find broken links on https://example.com/:
```

Comment:
Perhaps this will read better?
```suggestion
- Find broken links by downloading and analysing a webpage:
```

Reply:
It will not only analyse this webpage, but will also follow links to same domain and analyse every page it finds.